nlmod/gwt/gwt.py

```diff
@@ -0,0 +1,422 @@
+import logging
+import numbers
+
+import flopy
+import xarray as xr
+
+from ..dims import grid
+from ..gwf.gwf import _dis, _disv, _set_record
+
+logger = logging.getLogger(__name__)
+
+
+def _get_var_from_ds_attr(ds, varname, attr=None, var=None, warn=True):
```

Comment:
It took me a while to understand what this function does. I think a better name would be `_get_value_from_ds_attr`, curious what you think about that?

Is the idea that this is a general function that will be used in the future in gwf?

Reply:
Good suggestion, I agree about the name change. And yes, this was a first attempt at a more general approach for getting data from the model dataset in our model building functions.